nmmo/entity/player.py

```diff
@@ -31,9 +31,9 @@ def __init__(self, realm, pos, agent, color, pop):
       self.gold.update(1)
 
     self.diary  = None
-    tasks = realm.config.TASKS
-    if tasks:
-      self.diary = Diary(self, tasks)
+    # tasks = realm.config.TASKS
```

Comment:
should we just delete this? Or add a comment for why this is commented out

Reply:
I will also clean this part of the code in this iteration.

Reply:
Seems like we can just blow those away, they aren't being used for anything anymore

Reply:
I will clean these after making event-logging work.